meta-lmp-base/classes/lmp.bbclass

```diff
@@ -47,53 +51,61 @@ IMAGE_CMD:ostree:append () {
 
 run_fiotool_cmd () {
 	if [ -n "${SOTA_PACKED_CREDENTIALS}" ]; then
-                if [ -e "${SOTA_PACKED_CREDENTIALS}" ]; then
+		if [ -e "${SOTA_PACKED_CREDENTIALS}" ]; then
+			if [ -n "${1}" ] && [ -x "${1}" ]; then
+				cmd="${1}"
+			else
+				cmd="${2}"
+			fi
+			bbplain "Pushing/checking an ostree repo, cmd: ${cmd}"
 			if [ "${OSTREE_API_VERSION}" = "v2" ]; then
-				"${1}" -repo "${OSTREE_REPO}" -creds "${SOTA_PACKED_CREDENTIALS}" -api-version "${OSTREE_API_VERSION}" -cor-id "${GARAGE_TARGET_NAME}-${GARAGE_TARGET_VERSION}"
+				"${cmd}" -repo "${OSTREE_REPO}" -creds "${SOTA_PACKED_CREDENTIALS}" -api-version "${OSTREE_API_VERSION}" -cor-id "${GARAGE_TARGET_NAME}-${GARAGE_TARGET_VERSION}"
 			else
-				"${1}" -repo "${OSTREE_REPO}" -creds "${SOTA_PACKED_CREDENTIALS}"
+				"${cmd}" -repo "${OSTREE_REPO}" -creds "${SOTA_PACKED_CREDENTIALS}"
 			fi
-                else
-                        bbwarn "SOTA_PACKED_CREDENTIALS file does not exist."
-                fi
-        else
-                bbwarn "SOTA_PACKED_CREDENTIALS not set. Please add SOTA_PACKED_CREDENTIALS."
-        fi
+		else
+			bbwarn "SOTA_PACKED_CREDENTIALS file does not exist."
+		fi
+	else
+		bbwarn "SOTA_PACKED_CREDENTIALS not set. Please add SOTA_PACKED_CREDENTIALS."
+	fi
```

Comment:
Please also fix this indentation issues in a separated patch/commit.

Reply:
Done.